### PR TITLE
Feature/dataflow corrections

### PR DESCRIPTION
**Pull request by SMK1085**

### CODE REVIEW

**Comment by unity** [comment]
This won’t be enough to fix the dashboard issue. We need to refresh all the manifests - will hotfix the dashboard

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by unity** [comment]
while you're in it, can you add a `setIfNull` to the account.name if we handle_accounts = true ?

**Comment by SMK1085** [comment]
Adding the `setIfNull` to accounts @unity

File: server/clearbit/mapping.js [hunk 1]
```diff
@@ -67,5 +67,18 @@ export default function getUserTraitsFromPerson(
     {}
   );
 
+  // Overcome the value:key restrictions and add last_name and first_name to top level traits
```
Comment (unity) [inline server/clearbit/mapping.js]: Probably better to use the multiple-destination syntax in the mapper directly no ? : 
https://www.npmjs.com/package/object-mapper#string

**Comment by unity** [comment]
@SMK1085 we only setifNull the domain (https://github.com/hull-ships/hull-clearbit/blob/develop/server/clearbit.js) - I was referring to doing the same for the Name

File: server/clearbit/mapping.js (hunk 1 above)
Comment (unity) [reply server/clearbit/mapping.js]: @michaloo this seems like it could be improved with the above link

**Pull request merged**